### ISSUE TITLE
Import error when upgrading to jax 0.3.18

### ISSUE THREAD

**Issue by ASEM000** [deepmind/dm_pix#49]
Hi, 

`jax` version = 0.3.18
I am faced with the following error when importing `dm_pix`

I suspect it's due to the removal of `._src` since `jax` 0.3.18  https://github.com/google/jax/releases/tag/jax-v0.3.18 

```python
----> [1](vscode-notebook-cell:/Users/asem/serket/test.ipynb#Y121sZmlsZQ%3D%3D?line=0) import dm_pix

File ~/miniforge3/envs/dev-jax15/lib/python3.10/site-packages/dm_pix/__init__.py:16, in <module>
      1 # Copyright 2020 DeepMind Technologies Limited. All Rights Reserved.
      2 #
      3 # Licensed under the Apache License, Version 2.0 (the "License");
   (...)
     12 # See the License for the specific language governing permissions and
     13 # limitations under the License.
     14 """PIX public APIs."""
---> 16 from dm_pix._src import augment
     17 from dm_pix._src import color_conversion
     18 from dm_pix._src import depth_and_space

File ~/miniforge3/envs/dev-jax15/lib/python3.10/site-packages/dm_pix/_src/augment.py:25, in <module>
     22 import functools
     23 from typing import Callable, Sequence, Tuple, Union
---> 25 import chex
     26 from dm_pix._src import color_conversion
     27 from dm_pix._src import interpolation

File ~/miniforge3/envs/dev-jax15/lib/python3.10/site-packages/chex/__init__.py:17, in <module>
      1 # Copyright 2020 DeepMind Technologies Limited. All Rights Reserved.
      2 #
      3 # Licensed under the Apache License, Version 2.0 (the "License");
   (...)
     13 # limitations under the License.
     14 # ==============================================================================
     15 """Chex: Testing made fun, in JAX!"""
---> 17 from chex._src.asserts import assert_axis_dimension
     18 from chex._src.asserts import assert_axis_dimension_comparator
     19 from chex._src.asserts import assert_axis_dimension_gt

File ~/miniforge3/envs/dev-jax15/lib/python3.10/site-packages/chex/_src/asserts.py:26, in <module>
     23 import unittest
     24 from unittest import mock
---> 26 from chex._src import asserts_internal as _ai
     27 from chex._src import pytypes
     28 import jax

File ~/miniforge3/envs/dev-jax15/lib/python3.10/site-packages/chex/_src/asserts_internal.py:32, in <module>
     29 from typing import Any, Sequence, Union, Callable, Optional, Set, Tuple, Type
     31 from absl import logging
---> 32 from chex._src import pytypes
     33 import jax
     34 import jax.numpy as jnp

File ~/miniforge3/envs/dev-jax15/lib/python3.10/site-packages/chex/_src/pytypes.py:44, in <module>
     40 Device = jax.lib.xla_extension.Device
     42 ArrayTree = Union[Array, Iterable['ArrayTree'], Mapping[Any, 'ArrayTree']]
---> 44 ArrayDType = jax._src.numpy.lax_numpy._ScalarMeta

AttributeError: module 'jax' has no attribute '_src'
```

**Comment by claudiofantacci** [comment]
Hi @ASEM000, seems like there's a little hiccup with the Jax version and Chex there. I'll ask if a new version of Chex is going to be released soon to fix this issue with Jax.

If you have pip installed pix it might be that Jax has been installe dtransitively by Chex. The best thing to do is actually to install Jax befrehand (and maybe even with CUDA) and then install everything else. Anyway, thanks for flagging!

**Comment by hbq1** [comment]
Hi @ASEM000, the fix is already integrated into [chex v0.1.5](https://github.com/deepmind/chex/releases/tag/v0.1.5). 
You might need to update your chex version to fix the error that you're getting.
